modules/Emails/PopupDocuments.php

```diff
@@ -1,5 +1,4 @@
 <?php
-//_pp($_REQUEST);
 if(!defined('sugarEntry') || !sugarEntry) die('Not A Valid Entry Point');
```

Comment:
Coding Standards: Please update license file and move the sugar entry guard below the license file.

See: https://suitecrm.com/wiki/index.php/Coding_Standards